eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -1,41 +1,71 @@
+# Deploys resources to a cloud type specified by the variable (not parameter)
+# 'CloudType'. Use this as part of a matrix to deploy resources to a particular
+# cloud instance.
```

Comment:
Why not make CloudType a parameter to this template? While you could use the variable it seems like it would make things a little easier and more explicit if it were a parameter.